lib/bootloader_setup.pm

```diff
@@ -1022,6 +1022,7 @@ sub grep_grub_cmdline_settings {
     change_grub_config($old, $new [, $search ] [, $modifiers ], [, $update_grub ]);
 
 Replace C<$old> with C<$new> in /etc/default/grub, using sed.
+C<$new> will be keep '' null string when it is undef.
```

Comment:
I'd define it as: `C<$new>, default empty string`

Reply:
done